.github/workflows/ci.yml

```diff
@@ -24,7 +24,7 @@ jobs:
                 backend/bin/Release/tx_verifier
             release_command: 'npm run clean_build:nix && npm run release'
           - name: ubuntu
-            os: ubuntu-16.04
+            os: ubuntu-18.04
```

Comment:
We had some users running older Debian instances, which is why we built with 16.04. We're probably good to update this to 18.04 now, but you should check with the channel first, if you haven't already.

Reply:
Ubuntu 16.04 environment was removed on September 20, 2021 https://github.com/actions/virtual-environments/issues/3287

Reply:
I guess we don't really have a choice then. You'll want to mention that in the next release announcement.